crates/next-api/src/app.rs

```diff
@@ -831,19 +831,18 @@ impl AppProject {
                     } = &*find_server_entries(*rsc_entry).await?;
 
                     let graph = SingleModuleGraph::new_with_entries_visited(
-                        server_utils
-                            .iter()
-                            .map(|m| **m)
-                            .chain(extra_entries)
-                            .collect(),
+                        vec![(
+                            server_utils.iter().copied().chain(extra_entries).collect(),
+                            ChunkGroupType::Entry,
+                        )],
```

Comment:
Technically they would be `Shared`.
Just we allow ChunkingType::Shared to have `Option<ChunkGroupType>` to optionally create entry chunk groups too?